app/views/sessions/new.html.erb

```diff
@@ -10,4 +10,6 @@
       <%= f.submit "Login", class: "btn btn-danger" %>
     </div>
   <% end %>
+  <!-- Bootstrap JS -->
```

Comment:
this was addressed in another PR: bootstrap and other JS should ONLY be loaded once, in the common layout(s), not in individual views